pkgs/applications/plasma-mobile/srcs.nix

```diff
@@ -4,195 +4,187 @@
+      sha256 = "0n9g3jf71lpcwlwr1ia1i6kh0kiv2vizssni1df3cwskd7hivn7h";
+      name = "plasma-dialer-23.01.0.tar.xz";
     };
   };
   plasma-phonebook = {
-    version = "22.11";
+    version = "23.01.0";
     src = fetchurl {
-      url = "${mirror}/stable/plasma-mobile/22.11/plasma-phonebook-22.11.tar.xz";
-      sha256 = "02sj6v18w63ixy0c93fm4j9n4q98497qrb509q84618gry4cazly";
-      name = "plasma-phonebook-22.11.tar.xz";
+      url = "${mirror}/stable/plasma-mobile/23.01.0/plasma-phonebook-23.01.0.tar.xz";
+      sha256 = "0pprfgims9h6g1xw48w66bqkmqp7qp7gkc7k4zbj3lgmmnwh6z9i";
+      name = "plasma-phonebook-23.01.0.tar.xz";
     };
   };
   plasma-settings = {
-    version = "22.11";
+    version = "23.01.0";
     src = fetchurl {
-      url = "${mirror}/stable/plasma-mobile/22.11/plasma-settings-22.11.tar.xz";
-      sha256 = "0pg71plcbn4zkwxwcvay9swaa5xn3mnljdpnh4i8f86x2pa1pq5j";
-      name = "plasma-settings-22.11.tar.xz";
+      url = "${mirror}/stable/plasma-mobile/23.01.0/plasma-settings-23.01.0.tar.xz";
+      sha256 = "0fz0zjyh15ma6kv79pm0gc92iz6gikxyr4afkyzabzk9559dv1p7";
+      name = "plasma-settings-23.01.0.tar.xz";
     };
   };
   plasmatube = {
-    version = "22.11";
+    version = "23.01.0";
     src = fetchurl {
-      url = "${mirror}/stable/plasma-mobile/22.11/plasmatube-22.11.tar.xz";
-      sha256 = "0vgxvyw4yl91kaj9cnjf455mpj4a5rh5f1z0in8k9p2jbs8wmjil";
-      name = "plasmatube-22.11.tar.xz";
+      url = "${mirror}/stable/plasma-mobile/23.01.0/plasmatube-23.01.0.tar.xz";
+      sha256 = "06hwa1m6gaacjmcyssa63vw43cgx096x9aj87rv1z9k9qsv2qgfj";
+      name = "plasmatube-23.01.0.tar.xz";
     };
   };
   qmlkonsole = {
-    version = "22.11";
+    version = "23.01.0";
     src = fetchurl {
-      url = "${mirror}/stable/plasma-mobile/22.11/qmlkonsole-22.11.tar.xz";
-      sha256 = "03hwyrl6rb7yidw0nadis6csb6snpfv137j46rbpl35vrqrnrh5i";
-      name = "qmlkonsole-22.11.tar.xz";
+      url = "${mirror}/stable/plasma-mobile/23.01.0/qmlkonsole-23.01.0.tar.xz";
+      sha256 = "07h2pkz9h5lm232jasf6znf0i2p5c5qxnjac1d51qf8rfm0fp1q3";
+      name = "qmlkonsole-23.01.0.tar.xz";
     };
   };
   spacebar = {
-    version = "22.11";
+    version = "23.01.0";
     src = fetchurl {
-      url = "${mirror}/stable/plasma-mobile/22.11/spacebar-22.11.tar.xz";
-      sha256 = "1q0nyfvssiz5i19wyk4pvazax3vvk945cmknw4l51mpcc3hnc1v5";
-      name = "spacebar-22.11.tar.xz";
+      url = "${mirror}/stable/plasma-mobile/23.01.0/spacebar-23.01.0.tar.xz";
+      sha256 = "0srwvbnb80cgxkjjq2ch1lrbki7m169i86hig41h368wr9nmjnsl";
+      name = "spacebar-23.01.0.tar.xz";
     };
   };
   telly-skout = {
-    version = "22.11";
+    version = "23.01.0";
     src = fetchurl {
-      url = "${mirror}/stable/plasma-mobile/22.11/telly-skout-22.11.tar.xz";
-      sha256 = "0sfdia4jwcv24z5d3qykqv7ncr2c5v7ywq471nggwpcr22bwzn6x";
-      name = "telly-skout-22.11.tar.xz";
-    };
-  };
-  tokodon = {
```

Comment:
[you have nothing to do, noting my frustration]

After taking time to integrate "correctly" in #170466, it seems tokodon is now its own project handled out of gear... *sigh*.

 - https://download.kde.org/stable/tokodon/